### DISCUSSION TITLE
QUASAR_PUBLIC_FOLDER is no in typescript definitions

### DISCUSSION THREAD

**Discussion by mlilback** [quasarframework/quasar#16191] [CLI - Electron mode] [answered]
The [documentation](https://quasar.dev/quasar-cli-vite/developing-electron-apps/electron-accessing-files#accessing-the-public-folder) references a QUASAR_PUBLIC_FOLDER constant, but it is not defined in the `src-electron/electron-env.d.ts` that includes `QUASAR_ELECTRON_PRELOAD`.

I would have filed an issue or submitted a pull request, but both seem impossibly complicated for an issue like this. Requiring a URL to a reproducible case for a missing typescript definition seems to be way too much (especially since instructions say any issue without one will be automatically rejected, and Electron code can't be put on jsFiddle).

in case someone else runs into this problem, you can add the following to your project and it will all work:
```ts
// eslint-disable-next-line @typescript-eslint/no-namespace, @typescript-eslint/no-unused-vars
declare namespace NodeJS {
  interface ProcessEnv {
    QUASAR_PUBLIC_FOLDER: string
  }
}
```

**Comment by rstoenescu** [comment, marked as answer]
Thanks for catching this. Fixed in next version and a spawn of the Electron mode will have it by default.
